### ISSUE TITLE
Push running container to registry.

### ISSUE THREAD

**Issue by burtsevyg** [fabric8io/docker-maven-plugin#787]
Is there a possibility to push image which create from currantly running docker container?

**Comment by rhuss** [comment]
You mean to commit the container (after it is stopped) to create an image, and this image pushed to a registry?

No, that's not possible with dmp. What is your use case?

**Comment by burtsevyg** [comment]
Yes, you are right. My case is oracle db for test purpose.

**Comment by af6140** [comment]
Yeah, i think i kind of need this feature too for building oracle db.
